### ISSUE TITLE
Can't give nginx rights to read unix socket

### ISSUE THREAD

**Issue by jney** [pgte/fugue#11]
Hello,

I'm trying to configure nginx and fugue, but nginx doesn't have permission to read socket

```
srwx------ 1 www  www     0 2011-04-12 21:10 fugue.sock
srwxrwxrwx 1 root root    0 2011-04-12 20:48 unicorn.sock
```

So i tried to launch it as `nobody` :

```
fugue.start(app, '/tmp/www/sockets/fugue.sock', 4, {uid: 65534, gid: 65534});
```

but the owner of the socket is still the launcher of the program, current user or root when launched with `sudo`

How can i change this ?

**Comment by pgte** [comment]
Delete the file and try setting umask on your app before calling fugue.start

**Comment by jney** [comment]
hi pedro,

i tried :

```
var oldUmask = process.umask(0000);
fugue.start(app, '/tmp/www/sockets/fugue.sock', 4, {started: function () {
  process.umask(oldUmask);
}});
```

And :

```
umask 000 && node app.js
```

And nothing changed. I cannot figure out why

**Comment by pgte** [comment]
Ok. I just confirmed, it's an issue on fugue. Will work on it ASAP. Thanks for the report.

**Comment by jney** [comment]
ok, thank you for answering so fast

**Comment by pgte** [comment]
Should be fixed on release v0.1.2.
